Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN mkdir -p /usr/local/lib/bats/bats-support \
 
 # Install bats-assert
 RUN mkdir -p /usr/local/lib/bats/bats-assert \
-    && curl -sSL https://github.com/bats-core/bats-assert/archive/v0.3.0.tar.gz -o /tmp/bats-assert.tgz \
+    && curl -sSL https://github.com/bats-core/bats-assert/archive/v2.1.0.tar.gz -o /tmp/bats-assert.tgz \
```

Comment:
I don't have a better list, just whatever is in the [release notes of bats-core/bats-assert](https://github.com/bats-core/bats-assert/releases).

After all the changes, it might warrant a major version change, if you and everyone else thinks it's appropriate.